.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,121 @@
+            const totalTrackedTasks = parseInt(process.env.TOTAL_TRACKED_TASKS, 10)
+
+            const parseTasks = text => {
+              const tasks = []
+              text.split('\n').forEach(line => {
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[ \]\s+\S+/.test(line)) {
+                  tasks.push('')
+                }
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[[xX]\]\s+\S+/.test(line)) {
+                  tasks.push('☑')
+                }
+              })
+              return tasks
+            }
+
+            const parseLinks = text => {
+              const links = {}
+              text.split('\n').forEach(line => {
+                const linkMap = {
+                  draft: /^\[~draft-doc\]:\s*(?<link>\S+)/,
+                  sql: /^\[~sql-dir\]:\s*(?<link>\S+)/,
+                  results: /^\[~results-sheet\]:\s*(?<link>\S+)/
+                }
+                for (const [k, v] of Object.entries(linkMap)) {
+                  const m = line.match(v)
+                  if (m) {
+                    links[k] = m.groups.link
+                  }
+                }
+              })
+              return links
+            }
+
+            const parseTeam = text => {
+              const team = {}
+              text.split('\n').forEach(line => {
+                if (/\|\s*\[Sheet\]\[~results-sheet\]\s*\|/.test(line)) {
+                  const [, authors, reviewers, analysts] = line.split('|')
+                  team.authors = [...authors.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.reviewers = [...reviewers.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.analysts = [...analysts.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                }
+              })
+              return team
+            }
+
+            const formatLinks = links => {
+              const icons = {
+                draft: '📄',
+                sql: '🔍',
+                results: '📊'
+              }
+              return Object.entries(links).filter(l => l[1] && l[1] != '#').map(l => `[${icons[l[0]] || l[0]}](${l[1]})`).join(' ')
+            }
+
+            const formatTeam = team => {
+              return Object.entries(team).map(m => m[1].length).join('/')
+            }
+
+            const taskIssues = await github.paginate(github.issues.listForRepo, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              direction: 'asc',
+              labels: filterLabel
+            })
+
+            let report = ''
+            for (const issue of taskIssues) {
+              const tasksStatus = parseTasks(issue.body)
+              const tasksCount = tasksStatus.length
+              if(tasksCount != totalTrackedTasks) {
+                core.info(`Skipping issue #${issue.number} with ${tasksCount} tasks`)
+                continue
+              }
```

I actually think it's worth keeping that debugging in:

```suggestion
              } else {
                core.info(`Adding issue #${issue.number} with ${tasksCount} tasks`)  
              }  
```

---

Yes, I have added more debugging messages now, but the one I removed was logically irrelevant.